packages/website/components/page-templates/package-home.js

```diff
@@ -48,26 +50,42 @@ Header.propTypes = {
   heading: PropTypes.string.isRequired,
 };
 
-const PackageHome = ({ data, children }) => (
-  <NavigationWrapper
-    navContent={() => (
-      <PackageNavContent packageId={data.id} packageName={data.packageName} />
-    )}
+const MissingReadmeBanner = () => (
+  <Banner
+    icon={<WarningIcon label="Warning icon" secondaryColor="inherit" />}
+    isOpen
   >
-    <Wrapper>
-      <Header id={data.id} heading={titleCase(data.packageName)} />
-      <Description>{data.description}</Description>
-      <PackageMetaData
-        id={data.id}
-        version={data.version}
-        maintainers={data.maintainers}
-        repository={data.repository}
-      />
-      {children}
-    </Wrapper>
-  </NavigationWrapper>
+    There is no README for this package. This warning is only visible in dev
```

Comment:
Maybe we can re word this to be more optimistic by explaining how adding a readme will improve your docs and how you can use mdx to make rich/interactive readmes for even more bang

Reply:
Agree on the messaging